pkg/resources/nodeagent/daemonset.go

```diff
@@ -75,6 +75,10 @@ func (r *Reconciler) daemonSet() runtime.Object {
 									Name:  "Trust_Domain",
```

Comment:
Why are you keeping this one?

Reply:
I left it there so the operator to be backward compatible.